pkgs/by-name/nu/nu_scripts/package.nix

```
@@ -7,13 +7,13 @@
 
 stdenvNoCC.mkDerivation rec {
```

If `src.repo` is set literally, `rec` can just be dropped. Otherwise I’d suggest to use `finalAttrs`.

```suggestion
stdenvNoCC.mkDerivation {
```

---

Done